server/src/main/java/net/cryptic_game/backend/server/server/websocket/endpoints/WebSocketDaemonEndpoints.java

```diff
@@ -46,9 +46,9 @@ public ApiResponse send(final ApiClient client,
 
         App.addTimeout(App.getInstance().getConfig().getResponseTimeout() * 1000, () -> {
             if (daemonHandler.isRequstOpen(requestTag)) {
-                daemonHandler.respondToClient(JsonBuilder.anJSON()
-                        .add("tag", requestTag)
-                        .add("info", JsonBuilder.anJSON(ApiResponseType.BAD_GATEWAY.serialize())
+                daemonHandler.respondToClient(JsonBuilder
+                        .create("tag", requestTag)
+                        .add("info", JsonBuilder.create(ApiResponseType.BAD_GATEWAY.serialize())
```

Comment:
you dont need `serialize()`

Reply:
> 
> 
> you dont need `serialize()`

I do need `serialize()` as `JsonBuilder.create()` requires a `JsonObject` not an `ApiResponseType`.